sockeye/decoder.py

```diff
@@ -163,14 +171,18 @@ def __init__(self,
         # Decoder parameters
         # RNN init state parameters
         self._create_layer_parameters()
+
         # Hidden state parameters
         self.hidden_w = mx.sym.Variable("%shidden_weight" % prefix)
         self.hidden_b = mx.sym.Variable("%shidden_bias" % prefix)
+        self.hidden_norm = LayerNormalization(self.num_hidden,
+                                              prefix="%shidden_weight_norm" % prefix) if self.lnorm else None
```

Comment:
no, why would it be? prefix is "decoder_".
I can rename to "decoder_hidden_norm" if you dont want the weight in there.